test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -744,6 +744,21 @@ ConstructorTestClass(int i, bool b)
         $res.CompletionMatches[0].CompletionText | Should -BeExactly '$TestVar1'
     }
 
+    It "Should complete Verb parameter for '<TextInput>'" -TestCases @(
+        @{ TextInput = 'Get-Verb -Verb '; ExpectedCount = 100 }
+        @{ TextInput = 'Get-Verb -Verb Re'; ExpectedCount = 15 }
+        @{ TextInput = 'Get-Verb -Verb Ex'; ExpectedCount = 3 }
+        @{ TextInput = 'Get-Verb -Verb Conv'; ExpectedCount = 3 }
+        @{ TextInput = 'Get-Command -Verb '; ExpectedCount = 100 }
+        @{ TextInput = 'Get-Command -Verb Re'; ExpectedCount = 15 }
+        @{ TextInput = 'Get-Command -Verb Ex'; ExpectedCount = 3 }
+        @{ TextInput = 'Get-Command -Verb Conv'; ExpectedCount = 3 }
+    ) {
+        param($TextInput, $ExpectedCount)
+        $res = TabExpansion2 -inputScript $TextInput -cursorColumn $TextInput.Length
+        $res.CompletionMatches | Should -HaveCount $ExpectedCount
```

Comment:
It is not reliable tests. Since verb types are public we could do more  specific checks. (Build arrays from the types and compare with completion results.)

Reply:
@iSazonov Updated tests to be more explicit with completion text.